src/parsec_service/install_parsec_linux.md

```diff
@@ -30,6 +32,28 @@ Copy and adapt the [configuration](configuration.md) you want to use.
 cp parsec/config.toml config.toml
```

Comment:
Should the config file be in `/tmp/parsec/` with permissions `700`?

Reply:
Actually, that folder might be volatile 😞

Reply:
Yes! There are other permanent locations where the different files could go, that are used by other services, but the advantage of using the `parsec` user home folder and `/tmp` is that the installation steps rely less on using `sudo`.